tests/test_basic_splitter.py

```diff
@@ -0,0 +1,83 @@
+from draive import split_text
+
+
+def test_text_small_enough() -> None:
+    sample_text: str = "abcd efg hij"
+    result = split_text(
+        text=sample_text,
+        part_size=len(sample_text),
+        count_size=len
+    )
+    assert len(result) == 1
+    assert result[0] == sample_text
+
+def test_default_separator() -> None:
```

Comment:
I haven't paid much attention but it looks like we could use only a single test method being parametrized with a different scenarios (separators, expected outcome).